lib/src/markdown_processor.dart

```diff
@@ -145,7 +145,10 @@ final HtmlEscape htmlEscape = const HtmlEscape(HtmlEscapeMode.ELEMENT);
 final List<md.InlineSyntax> _markdown_syntaxes = [
   new _InlineCodeSyntax(),
   new _AutolinkWithoutScheme()
-];
+]..addAll(md.ExtensionSet.gitHub.inlineSyntaxes);
```

Comment:
See https://github.com/dart-lang/pub-dartlang-dart/blob/master/app/lib/shared/markdown.dart#L11

Not sure exactly what to do here. Ideally we'd all agree on how markdown is rendered.

https://github.com/dart-lang/pub-dartlang-dart/blob/8ae285f2b846b0e3bf1098162318f6c52eff5776/app/lib/shared/markdown.dart#L11

I've tried to move the markdown package towards GFM standard. But that doesn't include header links.

Just FYI that we've diverged here...

Reply:
Yes, I filed a bug related to this yesterday because markdown's README claims it does have header links for its github extension set: https://github.com/dart-lang/markdown/issues/185.

Dartdoc is always going to have at least a bit of customization, but hopefully we can get it to a point where it is purely additive..